stdlib/LinearAlgebra/test/dense.jl

```diff
@@ -1213,6 +1213,11 @@ end
     @test exp(log(A2)) ≈ A2
 end
 
+@testset "sqrt of empty Matrix of type $T" for T in [Int,Float32,Float64,ComplexF32,ComplexF64]
+    @test sqrt(Matrix{T}(undef, 0, 0)) == Matrix{T}(undef, 0, 0)
+    @test_throws DimensionMismatch sqrt(Matrix{T}(undef, 2, 3))
```

Comment:
I'd like to suggest to strengthen this a tiny bit.

```suggestion
    @test_throws DimensionMismatch sqrt(Matrix{T}(undef, 0, 3))
```